asaps/models.py

```diff
@@ -0,0 +1,132 @@
+from asnake.client import ASnakeClient
+import importlib
+import json
+import attr
+
+
+@attr.s
+class Client:
+    authclient = attr.ib
+
+    def createclient(secfile):
+        """Select secrets.py file for the appropriate instance."""
+        secfileExists = importlib.util.find_spec(secfile)
+        if secfileExists is not None:
+            secrets = __import__(secfile)
+        else:
+            secrets = __import__('secrets')
+        print('Editing ' + secfile + ' ' + secrets.baseURL)
+        authclient = ASnakeClient(baseurl=secrets.baseURL,
+                                  username=secrets.user,
+                                  password=secrets.password)
+        authclient.authorize()
+        setattr(Client, 'authclient', authclient)
```

Comment:
Change this line to `self.authclient = authclient`. This is how you would typically create an instance variable.